perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -192,6 +207,14 @@
 }
 AWS_S3_ENDPOINT_SUFFIX = '.amazonaws.com'
 
+# The amount of time in the future we ask all of our threads to sleep until.
+MULTISTREAM_BUFFER_TIME = 2.0
+
+# The delay between when we start reading and when we start writing is
+# SECS_PER_UPLOAD_KB * # KB uploaded per thread + READ_DELAY_SECS
+SECS_PER_UPLOAD_KB = 0.01
```

Comment:
If we use relative time to express read_start_time, then we won't need to get into this business of estimating how fast a provider can do uploads, which is quite fragile.

see more comments in the calculating write/read time logic